Source/Applications/MiMD/wwwroot/Scripts/TSX/MiMD/PRC-002/MeterOverview/MeterOverviewPage.tsx

```diff
@@ -66,10 +67,11 @@ const PRC002MeterOverviewPage = (props: { Roles: Array<MiMD.SecurityRoleName>, M
     const [showBaseConfig, setShowBaseConfig] = React.useState<boolean>(false);
 
     const [showFiles, setShowFiles] = React.useState<boolean>(false);
-    const [searchState, setSearchState] = React.useState<('Idle' | 'Loading'| 'Error')>('Idle');
+    const [searchState, setSearchState] = React.useState<('Idle' | 'Loading' | 'Error')>('Idle');
+    const [selectedID, setSelectedID] = React.useState<number>(1);
```

Comment:
This should make it so the top meter doesn't get highlighted as though it's selected upon navigating to the page.
```suggestion
    const [selectedID, setSelectedID] = React.useState<number>(null);
```